### ISSUE TITLE
toctree section headers (without index pages)

### ISSUE THREAD

**Issue by eharkins** [nextstrain/sphinx-theme#8]
As @jameshadfield [put it here](https://github.com/nextstrain/ncov/pull/709#issuecomment-904276901):
> I really want section headings (e.g. "Analysis" or "Getting started with analysis") to be simply headings in the sidebar, not pages in and of themselves. This holds for all our documentation on RTD, where we often have to create dummy pages with no real content.

According to @tsibley, this is possible: https://github.com/nextstrain/ncov/pull/709#issuecomment-904961475.

**Comment by jameshadfield** [comment]
This issue has been addressed for the [main docs project](https://docs.nextstrain.org/en/latest/) but not yet for the subprojects (e.g. [ncov](https://docs.nextstrain.org/projects/ncov/en/latest/analysis/index.html)). @tsibley was this achieved by https://github.com/nextstrain/docs.nextstrain.org/commit/7533335cf54266f810c347b9b2ab360e3417d090 ?

**Comment by tsibley** [comment]
Yeah, I hear you about the ideal. Going arbitrary levels deep like that isn't provided for by the RTD Sphinx theme. There's two things I see that we could do here:

1. Implement the desired sidebar behaviour ourselves by modifying the RTD navigation JS, with the knowledge that it'll be a divergence from both that as well as the way Sphinx models document hierarchies.

2. Accept the behaviour as good enough and leave it as-is, but instead of making section index pages like "Bioinformatics" a "dummy page with no real content", we could add actual content. :-) I think it'd be very useful to include [explanatory material](https://documentation.divio.com/explanation/) on index pages that discuss the topic and frame the subpages for a reader. We do this already to some degree on the ["Sharing analyses" index page](https://docs.nextstrain.org/en/latest/guides/share/index.html), for example.
